pkgs/development/python-modules/libretranslate/default.nix

```diff
@@ -83,6 +84,20 @@ buildPythonPackage rec {
 
   pythonImportsCheck = [ "libretranslate" ];
 
+  passthru = {
+    static-compressed = pkgs.runCommand "libretranslate-data-compressed" {
```

Comment:
This is used by nginx and should be probably be done in the module, but I've still not sold on the idea of nginx integration. Any gains from this change will be noticeable only on very high load, like 1000s of requests, but the libretranslate itself is really slow and it's going to be a bottleneck much sooner. The usage of mentioned proxy_cache (which could actually improve performance) will make this change obsolete as well.

Reply:
Any other web server can use this, too and having things cached from the module is not as easy as here.

> Any gains from this change will be noticeable only on very high load, like 1000s of requests,

No, serving compressed assets is worth it from the first request and not having to compress them on requests is faster. A simple 800kb css file can easily be compressed to 100kb which is a massive saving. Also compressing once on a higher level and then serve that until the next update which saves computation.

Reply:
I don't think it matters when libretranslate itself takes seconds or sometimes tens of seconds to do it's thing. It easir to maintain less code